plugins/webriq-payments/WebriQPayments.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import { withRouterHOC } from 'part:@sanity/base/router'
+import { useDocumentOperation, useEditState } from '@sanity/react-hooks'
+import {
+  Text,
+  Container,
+  Box,
+  Button,
+  Stack,
+  useToast
+} from '@sanity/ui'
+import { GrAdd} from 'react-icons/gr'
+import Dialog_Popup from './components/dialog'
+import ViewAccounts from './components/viewAccounts'
+import { encryptKey } from '../../utils/secureKey'
+
+// Sanity uses CSS modules for styling. We import a stylesheet and get an
+// object where the keys matches the class names defined in the CSS file and
+// the values are a unique, generated class name. This allows you to write CSS
+// with only your components in mind without any conflicting class names.
+// See https://github.com/css-modules/css-modules for more info.
+
+const WebriQPayments = () => {
+  const {useCallback, useState} = React
```

Comment:
These built-in React hooks can either be defined in line 1 or use the "React" default export name.